### PR TITLE
Install OpenJDK for Jazzer java fuzzer.

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

File: docker/base/Dockerfile [hunk 1]
```diff
@@ -57,6 +57,16 @@ RUN curl -sS https://releases.nixos.org/patchelf/patchelf-0.9/patchelf-0.9.tar.b
     ./configure --prefix=/usr && \
     make install
 
+# Install OpenJDK 15 for Jazzer (Java fuzzer).
```
Comment (jonathanmetzman) [inline docker/base/Dockerfile:60]: Maybe link to oss-fuzz/base-builder so it's clearer where this is copied from.

**Review by jonathanmetzman — APPROVED** [review]
LGTM
Thanks for doing this!

**Review by oliverchang — APPROVED** [review]
LGTM. how much size does this add?

File: docker/base/Dockerfile (hunk 1 above)
Comment (inferno-chromium) [reply docker/base/Dockerfile:60]: Done!

**Comment by inferno-chromium** [comment]
> LGTM. how much size does this add?

+191mb

**Pull request merged**